sdks/java/io/hadoop-input-format/pom.xml

```diff
@@ -96,7 +96,6 @@
         <dependency>
           <groupId>com.fasterxml.jackson.module</groupId>
           <artifactId>jackson-module-scala_2.11</artifactId>
-          <version>2.8.9</version>
           <scope>runtime</scope>
```

Comment:
Is the version here somehow 'inherited' from the top level pom? Or why don't we have it here?

Reply:
It is inherited from the root pom.xml here:
https://github.com/apache/beam/blob/b807847455e5fb1b2fbdd5285a2f696960ab754f/pom.xml#L1275

Reply:
Cool. SGTM.